CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
-* None.
+* Storage of Decimal128 properties has been optimised so that the individual values will take up 32 bits, 64 bits or 128 bits depending on what is needed. (PR [#6111]https://github.com/realm/realm-core/pull/6111))
```

Comment:
Should the list of sizes also include 0 bits?

Reply:
Like now?

Reply:
Will it only collapse to zero bits for all nulls, or also for all 0.0?

Reply:
We have to decide if we want to interpret zero width as null or 0.0. I think null is the most obvious choice.

Reply:
Can we use the array's context flag to determine if a width of 0 means nulls or zeros?